integration-tests/src/test/scala/tech/beshu/ror/integration/suites/CrossClusterCallsSuite.scala

```diff
@@ -94,22 +97,56 @@ class CrossClusterCallsSuite
   private lazy val user3SearchManager = new SearchManager(basicAuthClient("dev3", "test"), esVersionUsed)
   private lazy val user4SearchManager = new SearchManager(basicAuthClient("dev4", "test"), esVersionUsed)
   private lazy val user5SearchManager = new SearchManager(basicAuthClient("dev5", "test"), esVersionUsed)
+  private lazy val indexManager = new IndexManager(basicAuthClient("dev1", "test"), esVersionUsed)
 
   "A cluster _search for given index" should {
     "return 200 and allow user to its content" when {
       "user has permission to do so" when {
         "he queries local and remote indices" in eventually {
-          val result = user3SearchManager.search("etl1:etl*", "metrics*")
+          val result = user3SearchManager.search("private1:audit*", "metrics*")
           result should have statusCode 200
           result.searchHits.map(i => i("_index").str).toSet should be(
-            Set("metrics_monitoring_2020-03-26", "metrics_monitoring_2020-03-27", "etl1:etl_usage_2020-03-26", "etl1:etl_usage_2020-03-27")
+            Set("metrics_2020-03-26", "metrics_2020-03-27", "private1:audit_2020-03-26", "private1:audit_2020-03-27")
           )
         }
         "he queries remote indices only" in {
-          val result = user1SearchManager.search("etl2:test1_index")
+          val result = user1SearchManager.search("private2:test1_index")
           result should have statusCode 200
           result.searchHits.arr.size should be(2)
         }
+        "he queries remote data streams only by data stream name" excludeES (allEs7xBelowEs77x) in {
+          val result = user1SearchManager.search("private2:test1_ds")
+          result should have statusCode 200
+          result.searchHits.arr.size should be(3)
```

Comment:
TBH this is not a good approach (I'm talking based on my experience with ROR tests that passes locally but fail in the pipeline). 

When there are more than 3 hists, the test will fail here. For sure we will be interested in what the search result returned. But there is no such log so there should be a problem with the analysis. 

There are at least two solutions here:
1. The same what we verify here, will also be verified by the last assertion. So, this one can be skipped.
2. nicely wrapped scalatest's [Checkpoint](https://www.scalatest.org/scaladoc/3.1.2/org/scalatest/Checkpoints$Checkpoint.html)